test_conformance/images/clCopyImage/test_copy_generic.cpp

```diff
@@ -43,6 +58,26 @@ cl_mem create_image( cl_context context, cl_command_queue queue, BufferOwningPtr
     imageDesc.image_slice_pitch = gEnablePitch ? imageInfo->slicePitch : 0;
     imageDesc.num_mip_levels = gTestMipmaps ? imageInfo->num_mip_levels : 0;
 
+    cl_version version;
+    cl_device_id device;
+    {
+        cl_int err = clGetCommandQueueInfo(queue, CL_QUEUE_DEVICE,
+                                           sizeof(device), &device, nullptr);
+        if (err != CL_SUCCESS)
+        {
+            log_error("Error: Could not get CL_QUEUE_DEVICE from queue");
+            return nullptr;
+        }
+        err = clGetDeviceInfo(device, CL_DEVICE_NUMERIC_VERSION,
+                              sizeof(version), &version, nullptr);
```

Comment:
Using `CL_DEVICE_NUMERIC_VERSION` here means that these tests will require OpenCL 3.0.  Could we use the test harness function `get_device_cl_version()` instead, which will work for all OpenCL versions?

If so, since the PR description says "the change queries the version similar to clFillImage test binary," maybe we should update the clFillImage test also?